hidden-role-to-go-core/src/main/java/com/github/virtualgamers/hrtg/core/service/PartyService.java

```diff
@@ -0,0 +1,39 @@
+package com.github.virtualgamers.hrtg.core.service;
+
+import org.springframework.stereotype.Component;
+import com.github.virtualgamers.hrtg.core.model.Party;
+import com.github.virtualgamers.hrtg.core.model.User;;
+
+/**
+ * {@link PartyService} sets up a {@link Party}
+ *
+ * @author CCThomas
+ *
+ */
+@Component
+public interface PartyService {
+
+    /**
+     * Add {@link User} to a {@link Party} with partyId
+     *
+     * @param partyId for a {@link Party}
+     * @param user {@link User}
+     */
+    User addToParty(String partyId, String username);
```

Comment:
param username, also should be included with the top-level doc if the partyId is in there too?

Reply:
Originally I was passing a "User", and forgot to update "@param user {@link User}" to "@param username {@link String}" when I made the change